boskos/README.md

```diff
@@ -0,0 +1,51 @@
+# WIP boskos
+
+
+## Background
+[βοσκός](https://en.wiktionary.org/wiki/%CE%B2%CE%BF%CF%83%CE%BA%CF%8C%CF%82) - cattle-master in greek!
+
+boskos is a http service that allow k8s test jobs to run independently from GCP projects.
```

Comment:
What do we gain by developing, managing and maintaining a centralized http service?

It seems like we can accomplish most of this in a cooperative, decentralized way using a GCS object and the read-modify-write pattern, plus a replicated janitor service to clean up dirty projects.